src/lib.rs

```diff
@@ -1,7 +1,11 @@
 #![no_std]
 #![cfg_attr(target_arch = "xtensa", feature(asm_experimental_arch))]
-#![cfg_attr(feature = "defmt-espflash", feature(panic_info_message))]
+#![cfg_attr(feature = "defmt", feature(panic_info_message))]
```

Comment:
I didn't notice this feature before, do we really need/want to pull in a nightly feature for printing the panic message, couldn't we use `defmt::Display2Format` to print via defmt?

Reply:
I really would like to get rid of this! I'll have a look

Reply:
I removed it ... doesn't look great because the new-line is printed as `\n` but I guess it's fine for now